CHANGELOG.md

```diff
@@ -45,6 +46,8 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Fixed wrong translation % in F1-Menu when changing language (by @NickCloudAT)
 - Fixed disguiser breaking UI on hot reload (by @TimGoll)
 - Fixed blurred box rendering for boxes not starting at `0,0` (by @TimGoll)
+- Optimized allocations by using global Vector / Angle when possible
```

Comment:
We should not do this as @EntranceJew noted that these seem to be mutated.

Reply:
I only removed  the `.` because it annoyed me :D

Reply:
oh nvm then